controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -264,14 +310,21 @@ func (r *OCSInitializationReconciler) Reconcile(ctx context.Context, request rec
 
 // SetupWithManager sets up a controller with a manager
 func (r *OCSInitializationReconciler) SetupWithManager(mgr ctrl.Manager) error {
+	cluster, err := cluster.New(mgr.GetConfig(), func(options *cluster.Options) {
+		options.Scheme = mgr.GetScheme()
+	})
```

Comment:
How about we fetch it in main and pass it to all controllers and it can be used in a storagecluster controller as well.